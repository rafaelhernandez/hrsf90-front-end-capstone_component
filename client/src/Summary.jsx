import React from 'react';

const Summary = (props) => (
  <div className="div-description-summary">
    {props.summary}
  </div>
);
export default Summary;
