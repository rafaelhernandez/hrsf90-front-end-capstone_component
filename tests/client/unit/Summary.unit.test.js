import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Summary from '../../../client/src/Summary.jsx';
import descriptionSample from '../../../db/data/description_sample.js';

configure({ adapter: new Adapter() });

describe('Summary', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Summary desc={descriptionSample.summary}></Summary>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
