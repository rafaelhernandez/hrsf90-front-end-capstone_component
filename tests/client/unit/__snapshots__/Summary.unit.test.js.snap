// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summary should render correctly 1`] = `
<div
  className="div-description-summary"
/>
`;
